unischedule/lib/presentation/calendar_page/calendar_page.dart

```diff
@@ -75,11 +76,16 @@ class _CalendarAppState extends State<CalendarApp> {
       Positioned(
         right: 11,
         bottom: 11 + MediaQuery.of(context).padding.bottom,
-        child: SvgPicture.asset(
-          'assets/icons/square-plus.svg',
-          width: 50,
-          height: 56,
-          color: Colors.white,
+        child: InkWell(
+          onTap: () {
+            GoRouter.of(context).go('/create-class');
+          },
+          child: SvgPicture.asset(
+            'assets/icons/square-plus.svg',
+            width: 50,
+            height: 56,
+            color: Colors.white,
+          ),
```

Comment:
Hey @iGotty ! I modified your file to add navigation support to the new view I developed. let me know what do you think!

Reply:
Thanks for adding navigation support to the calendar view! I just reviewed the changes and everything seems to be well integrated and in line with our existing conventions.